docs/browser.md

```diff
@@ -7,6 +7,10 @@ the desktop version.
 
 You can run the browser implementation of the extension in the following ways.
 
+## Pre-requisutes
+
+-   Install the `TypeScript + Webpack Problem Matchers` extension in the VS Code marketplace. This allows us to detect that webpack `watch` is done since it never actually terminates. It is utilized in `tasks.json`.
```

Comment:
I think "problem matchers" is already mentioned in CONTRIBUTING.md, or if not let's add it there. I think any prereqs should live there, rather than spreading that out in various docs.

Reply:
Updated our CONTRIBUTING.md doc and linked to that in this browser.md doc